andela/samurais-healthchecks
# #147272005 User should be able to set timeouts and grace values or days exceeding 30

#### What does this PR do?
User are able to set timeouts and grace values for days exceeding 30
#### Description of Task to be completed?

1. WHEN selecting a grace period for that job then I have the option of setting a grace period value of more than 30 days.
2. WHEN setting a timeout for that job then I have the option of setting a timeout value of more than 30 days.

#### How should this be manually tested?
Log in to healthchecks add a check then set a timeout and grace period you'll notice the slider 
has options to set values of up to a year, then click save.
#### Any background context you want to provide?
None
